### DISCUSSION TITLE
Do organizations cost

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#134370] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I have a GitHub Pro license/membership. I've been considering starting to use organizations. At work we have two organizations within our GitHub Enterprise. But I don't know if organizations cost anything extra to add to a GitHub Pro license.

Do they  cost more?

**Comment by BhathiyaPrasad** [comment]
when it comes to the teams and collaborators in private repositories, I think it costs 

if this message helps you to understand please mark as answer

**Comment by CleverLemming1337** [comment, marked as answer]
You can create a small organization for free. But if you want to use GitHub Pages, Rulesets or Wikis etc. in private repositories you have to upgrade your org. 

This is basically the same as a normal profile.

**Reply by rfalanga** [reply]
I don't know what comprises a small organization, but I've got enough to understand what's involved. Thank you for your response.
